packages/components/pie-notification/test/component/pie-notification.spec.ts

```diff
@@ -473,6 +473,53 @@ test.describe('PieNotification - Component tests', () => {
                 });
             });
         });
+        test.describe('position', () => {
+            test('should render by default the position prop as inline-content and the border radius must be 12px', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieNotification, {
+                    props: {},
+                });
+
+                // Act
+                const notification = page.locator(componentSelector);
+
+                // Assert
+                expect(notification).toBeVisible();
+                expect(notification).toHaveCSS('border-radius', '12px');
```

Comment:
These feel a bit more like visual tests to me. I'd be interested to know what other people think though.

Reply:
In my opinion, although it looks quite close to a visual test I believe this is suitable as a unit test since I am testing the behaviour of the component based on its inputs as well what is the default behaviour if no prop is passed. IMHO these unit tests and visual tests added in this PR complement each other and I don't see that as duplication.

Reply:
@thejfreitas I see your point and it's entirely valid. What I wonder is do we expect this test to fail but the visual test to pass? I think the answer is probably not, and therefore I would personally prefer we leave this to visual testing.

Another issue is if the design token value changes then we have to update this test as we're referencing a raw value.

Reply:
That's my intention if somehow the value/token changes for whatever reason this will make sure that we need to update the token to a new one in this component. I had this scenario in the past in PIE React and as I said above IMHO they complement each other but I am ok to remove it.

@ashleynolan @xander-marjoram @jamieomaguire please react with 👍🏼 to keep or 👎🏼 to remove this test I am fine with both options.

Reply:
Hi @thejfreitas, sorry for the slow response. My understanding is that if the tokens change then we want consumers to update automatically. I'd be in favour of removing this test as the visual test should cover it.